src/sbt-test/sbt-sql/simple/project/plugins.sbt

```diff
@@ -0,0 +1 @@
+addSbtPlugin("org.xerial.sbt" % "sbt-sql" % "0.2-SNAPSHOT")
```

Comment:
Updating this plugin version will be cumbersome in near future. I'll add an script to update this.